composer.json

```diff
@@ -25,7 +25,8 @@
         "psr/cache":        "^1.0",
         "predis/predis":    "^1.0",
         "doctrine/cache":   "^1.3",
-        "league/flysystem": "^1.0"
+        "league/flysystem": "^1.0",
+        "symfony/options-resolver": "^3.0"
```

Comment:
yeah. 2.7 would actually be better. Same with the one below (src/Adapter/Chain/composer.json)